test/UnitTests/Microsoft.Testing.Extensions.UnitTests/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Microsoft.Testing.Extensions.UnitTests": {
       "commandName": "Project",
-      "commandLineArgs": "--treenode-filter /*/*/*/**"
+      "commandLineArgs": "--treenode-filter /*/*/*/** --output detailed"
```

Comment:
do you want to merge this one too?

Reply:
```suggestion
      "commandLineArgs": "--treenode-filter /*/*/*/**
```

Reply:
Let's do it on all test files as a separate PR. I think that's a good default.